plugins/modules/vyos_ospfv2.py

```diff
@@ -0,0 +1,1981 @@
+          neighbor_id:
+            description: Identity (number/IP address) of neighbor.
+            type: str 
+          poll_interval:
+            description: Seconds between dead neighbor polling interval.
+            type: int
+          priority:
+            description: Neighbor priority.
+            type: int       
+      parameters:
+        descriptions: OSPFv2 specific parameters.
+        type: dict
+        suboptions: 
+         abr_type: 
+           description: OSPFv2 ABR Type.
+           type: str
+           choices: ['cisco', 'ibm', 'shortcut', 'standard']
+         opaque_lsa:
+           description: Enable the Opaque-LSA capability (rfc2370).
+           type: bool
+         rfc1583_compatibility:
+           description: Enable rfc1583 criteria for handling AS external routes.
+           type: bool
+         router_id: 
+           description: Override the default router identifier.
+           type: str
+      passive_interface:
+        description: Suppress routing updates on an interface.
+        type: list
+      passive_interface_exclude:
+        description: Interface to exclude when using passive-interface default.
+        type: list
+      redistribute:  
+        description: Redistribute information from another routing protocol.
+        type: list
+        elements: dict
+        suboptions: 
+          route_type:
+            description: Route type to redistribute.
+            type: str
+            choices: ['bgp', 'connected', 'kernel', 'rip', 'static']
+          metric:
+            description: Metric for redistribution routes.
+            type: int
+          metric_type:
+            description: OSPFv2 Metric types.
+            type: int
+          route_map:
+            description: Route map references.
+            type: str 
+      
+      route_map:
+        description: Filter routes installed in local route map.       
+        type: list
+      timers:
+        description: Adjust routing timers.
+        type: dict
+        suboptions:
+          refresh:
+            description: Adjust refresh parameters.
+            type: dict
+            suboptions:
+              timers:
+                description: refresh timer.
+                type: int
+          throttle:
+            description: Throttling adaptive timers.
+            type: dict
+            suboptions:
+              spf:
+                description: OSPFv2 SPF timers.
+                type: dict
+                suboptions:
+                  delay:
+                    description: Delay (msec) from first change received till SPF calculation.
+                    type: int
+                  initial_holdtime:
+                    description: Initial hold time(msec) between consecutive SPF calculations.
+                    type: int
+                  max_holdtime:
+                    description: maximum hold time (sec).
+                    type: int
+  running_config:
+    description:
```

Comment:
The description for `running_config` is not correct at the moment. It is similar to what we have in https://docs.ansible.com/ansible/latest/modules/ios_config_module.html#parameter-running_config. But the RM's `running_config` is different from the *_config one. What do you think about updating it to - https://github.com/ansible-collections/cisco.nxos/pull/33/files#diff-8f4ac16689eeae1ccce530dea6d10c9bR51?